app/src/main/graphql/fragments.graphql

```diff
@@ -362,6 +362,32 @@ fragment shippingRule on ShippingRule {
     }
 }
 
+
+
+fragment ppoCard on Backing {
```

Comment:
As of right now, each card only has the backing in the response from graph, so im retrieving the information we need from it with this fragment. this is subject to change depending on how the rest of the response is structured